### PR TITLE
Portworx shared volumes article

### PR DESCRIPTION
**Pull request by benbacon**

### CODE REVIEW

File: articles/openshift/oshift-how-use-portworx-shared-volumes.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+# How to use Portworx shared volumes
+
+## Overview
+
+Portworx is a cloud-native storage solution that is now available as an integration with our OpenShift offering. This article describes how to create shared volumes.
+
+### Intended audience
+
+This article assumes you have access to a Portworx integrated OpenShift 3.11 cluster and that you have cluster-admin rights. It also assumes familiarity with `oc`, the OpenShift command-line client.
```
Comment (stevemul) [inline articles/openshift/oshift-how-use-portworx-shared-volumes.md]: Should we specify 3.11 or greater? That way we future proof it and only have to validate that this hasn't changed how these features are used (which I don't think a new version of OpenShift would change).

File: articles/openshift/oshift-how-use-portworx-shared-volumes.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+### Intended audience
+
+This article assumes you have access to a Portworx integrated OpenShift 3.11 cluster and that you have cluster-admin rights. It also assumes familiarity with `oc`, the OpenShift command-line client. 
+
+If you're interested in a free 30 day trial of Portworx, raise a Service Request via the [My Calls](https://portal.skyscapecloud.com/support/ivanti) section of the UKCloud Portal.
+
+## Shared volumes
+
+Portworx enables persistent volumes claims to be mounted to multiple containers across multiple nodes enabling ease of pod-scaling.
```
Comment (stevemul) [inline articles/openshift/oshift-how-use-portworx-shared-volumes.md]:
```suggestion
Portworx enables persistent volumes to be mounted to multiple containers across multiple nodes.
```
If you want to talk more specifically about claims here you can, but in general what we're mounting is the PV not the PVC. 
Also is it enabling ease of pod scaling? For me it's enabling volumes to be accessible across multiple pods - there are lots of complications that come from a shared filesystem potentially so I'm not sure its ease of pod-scaling so much.

File: articles/openshift/oshift-how-use-portworx-shared-volumes.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+If you want to use different attributes to the pre-provisioned storage classes, include the following values within the spec:
+
+```none
+provisioner: com.openstorage.pxd
+parameters:
+	shared: "true"
+```
+
+> [!IMPORTANT]
+> Within OpenShift, Portworx shared volumes make use of the Container Storage Interface (CSI) via the `com.openstorage.pxd` provisioner. OpenShift 3.11 uses v0.3 of the CSI spec, which is a beta release, so use of shared volumes is not recommended within a production environment.
```
Comment (stevemul) [inline articles/openshift/oshift-how-use-portworx-shared-volumes.md]: This is good - important point.

File: articles/openshift/oshift-how-use-portworx-shared-volumes.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+              - "-c"
+              - >
+                while true; do
+                  date >> /pod-data/sharedfile;
+                  echo Hello from the second pod >> /pod-data/sharedfile;
+                  sleep 1;
+                done
+    ```
+
+2. You can now create the deployments with te command: `oc create -f sharedvolumedemo.yaml`, which should output the following:
```
Comment (stevemul) [inline articles/openshift/oshift-how-use-portworx-shared-volumes.md]:
```suggestion
2. You can now create the deployments with the command: `oc create -f sharedvolumedemo.yaml`, which should output the following:
```

File: articles/openshift/oshift-how-use-portworx-shared-volumes.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+            args:
+              - "-c"
+              - >
+                while true; do
+                  date >> /pod-data/sharedfile;
+                  echo Hello from the second pod >> /pod-data/sharedfile;
+                  sleep 1;
+                done
+    ```
+
```
Comment (stevemul) [inline articles/openshift/oshift-how-use-portworx-shared-volumes.md]: It'd be useful here to call out some of the details of the above - for example where we set anti-affinity and maybe a link to where more can be read about that.

File: articles/openshift/oshift-how-use-portworx-shared-volumes.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+```
+
+Once you've created the pvc, verify that its status shows as `Bound` to confirm it has been provisioned by Portworx:
+
+```none
+$ oc get pvc
+NAME             STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS            AGE
+pvc-shared   Bound     pvc-3654b857-efa1-11e9-8422-fa163e52fd0e   10Gi       RWX            portworx-repl1-shared   1m
+```
+
```
Comment (stevemul) [inline articles/openshift/oshift-how-use-portworx-shared-volumes.md]: Should we be pointing out and showing the PV that gets created when the PVC dynamically provisions the volume?

File: articles/openshift/oshift-how-use-portworx-shared-volumes.md (hunk 5 above)
Comment (benbacon) [reply articles/openshift/oshift-how-use-portworx-shared-volumes.md]: Is what I've added in commit c9d4284 what you had in mind?

**Pull request merged**